ci/builders/mac_unopt.json

```diff
@@ -223,6 +223,77 @@
                     "script": "flutter/testing/scenario_app/run_ios_tests.sh"
                 }
 
+            ]
+        },
+        {
+            "archives": [
```

Comment:
Since intel machines are limited and the test suite are the same, only add Apple silicon machine for testing.